alinka/widget/containers/main_body/content/settings/settings_tabs/schools_tab/__init__.py

```diff
@@ -14,3 +14,11 @@ def __init__(self, parent: QWidget):
 
         self.school_data_group = SchoolDataGroup(self)
         layout.addWidget(self.school_data_group)
+
+    @property
+    def is_valid(self) -> bool:
+        return True
+
+    @property
+    def error_message(self) -> str | None:
```

Comment:
I don’t get this concept. How it’s going to return string?

Reply:
This method should be fully implemented on next iterations. Goal is to have this two method `is_valid` and `error_message` on every container, which is supposed to  be validated. Not we have it to provide consisted `api`.

Reply:
Comment, and reference to ticket have been added.